.github/workflows/standalone.yml

```diff
@@ -102,7 +102,7 @@ jobs:
     steps:
     # osx signing based on https://melatonin.dev/blog/how-to-code-sign-and-notarize-macos-audio-plugins-in-ci/
     - name: Import Certificates (macOS)
-      uses: apple-actions/import-codesign-certs@63fff01cd422d4b7b855d40ca1e9d34d2de9427d  # v3
+      uses: apple-actions/import-codesign-certs@cfd6eb39a2c848ead8836bda6b56813585404ba7  # v5.0.0
```

Comment:
Not sure about  this one though.

Reply:
Yeah I am surprised. I thought Apple would refuse to sign. 😆 

Merge away! Thanks.